examples/sprite_camera_follow/main.rs

```diff
@@ -0,0 +1,182 @@
+        WindowEvent, ALPHA,
+    },
+    utils::application_root_dir,
+};
+
+#[derive(Default)]
+struct Player;
+
+impl Component for Player {
+    type Storage = NullStorage<Self>;
+}
+
+struct MovementSystem;
+
+impl<'s> System<'s> for MovementSystem {
+    type SystemData = (
+        ReadStorage<'s, Player>,
+        WriteStorage<'s, Transform>,
+        Read<'s, InputHandler<String, String>>,
+    );
+
+    fn run(&mut self, (_asdf, mut transforms, input): Self::SystemData) {
+        let x_move = input.axis_value("entity_x").unwrap();
+        let y_move = input.axis_value("entity_y").unwrap();
+
+        for (_, player) in (&_asdf, &mut transforms).join() {
+            player.translation.x += x_move as f32 * 5.0;
+            player.translation.y += y_move as f32 * 5.0;
+        }
+    }
+}
+
+fn load_sprite_sheet(world: &mut World) -> SpriteSheetHandle {
+    let res_path = format!(
+        "{}/examples/sprite_camera_follow/resources",
+        application_root_dir()
+    );
+    let texture_handle = {
+        let loader = world.read_resource::<Loader>();
+        let texture_storage = world.read_resource::<AssetStorage<Texture>>();
+        loader.load(
+            format!("{}/Circle.png", res_path),
+            PngFormat,
+            TextureMetadata::srgb_scale(),
+            (),
+            &texture_storage,
+        )
+    };
+    let texture_id = 0;
+    let mut material_texture_set = world.write_resource::<MaterialTextureSet>();
+    material_texture_set.insert(texture_id, texture_handle);
+
+    let loader = world.read_resource::<Loader>();
+    let sprite_sheet_store = world.read_resource::<AssetStorage<SpriteSheet>>();
+    loader.load(
+        format!("{}/Circle.ron", res_path),
+        SpriteSheetFormat,
+        texture_id,
+        (),
+        &sprite_sheet_store,
+    )
+}
+
+fn init_sprite(world: &mut World, sprite_sheet: &SpriteSheetHandle) -> Entity {
+    let mut transform = Transform::default();
+    transform.translation.x = 100.0;
+    transform.translation.y = 0.0;
+    let sprite = SpriteRender {
+        sprite_sheet: sprite_sheet.clone(),
+        sprite_number: 1,
+        flip_horizontal: false,
+        flip_vertical: false,
+    };
+    world.create_entity().with(transform).with(sprite).build()
+}
+
+fn init_player(world: &mut World, sprite_sheet: &SpriteSheetHandle) -> Entity {
+    let mut transform = Transform::default();
+    transform.translation.x = 0.0;
+    transform.translation.y = 0.0;
+    let sprite = SpriteRender {
+        sprite_sheet: sprite_sheet.clone(),
+        sprite_number: 0,
+        flip_horizontal: false,
+        flip_vertical: false,
+    };
+    world
+        .create_entity()
+        .with(transform)
+        .with(Player)
+        .with(sprite)
+        .build()
+}
+
+fn init_camera(world: &mut World, parent: Entity) {
+    let mut transform = Transform::default();
+    transform.translation.z = 1.0;
```

Comment:
I'll have to do some testing, but I thought that's how the `Parent` component worked? e.g. the `Camera` will add it's transform onto it's Parent's transform, since it isn't a `GlobalTransform`. I'm new to the project though so let me know if that isn't how it works. I'll test it later today otherwise.

Reply:
I'll confirm it later before I resolve